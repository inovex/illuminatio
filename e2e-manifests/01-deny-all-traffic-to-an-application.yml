apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: web
  name: web
  namespace: 01-deny-all
spec:
  replicas: 1
  selector:
    matchLabels:
      app: web
  template:
    metadata:
      namespace: 01-deny-all
      labels:
        app: web
    spec:
      containers:
      - image: nginx
        name: web
        ports:
        - containerPort: 80
          protocol: TCP
---
apiVersion: v1
kind: Service
metadata:
  namespace: 01-deny-all
  labels:
    app: web
  name: web
spec:
  ports:
  - port: 80
    protocol: TCP
    targetPort: 80
  selector:
    app: web
  type: ClusterIP
---
kind: NetworkPolicy
apiVersion: networking.k8s.io/v1
metadata:
  namespace: 01-deny-all
  name: web-deny-all
spec:
  podSelector:
    matchLabels:
      app: web
  ingress: []
