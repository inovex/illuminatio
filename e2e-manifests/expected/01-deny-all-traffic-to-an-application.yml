01-deny-all-traffic-to-an-application:app=web:
  01-deny-all-traffic-to-an-application:app=web:
    -TCP/*:
      success: true
    -UDP/*:
      success: true
